user_guide_src/source/libraries/email.rst

```diff
@@ -87,6 +87,32 @@ The settings used for the last successful send are available from the
 instance property ``$archive``. This is helpful for testing and debugging
 to determine that actual values at the time of the ``send()`` call.
 
+SSL versus TLS for SMTP Protocol
+================================
+
+To protect the username, password and email content while communicating with the SMTP server,
+encryption on the channel should be used used. Two different standards are widely deployed and
+it is important to understand the differences when trying to troubleshoot email sending
+issues.
+
+Most SMTP servers allow connections on ports 465 or 587 when submitting emails. (The
+original port 25 is seldom used because of many ISPs have blocking rules in place and
+since the communication is entirely in clear-text).
+
+The key difference is that port 465 expects the communication channel to be secured using TLS
+from the start as per RFC8314. A connection to port 587 allows clear-text connection and later
```

Comment:
probably add an external link to this rfc for cross reference.

Reply:
Thanks!